torchmetrics/functional/audio/si_sdr.py

```diff
@@ -0,0 +1,60 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+
+
+def si_sdr(target, estimate, zero_mean=False, EPS=1e-8):
```

Comment:
Is the `EPS` parameter relevant to the user?
If it is only because of numerical stability then it should probably be removed.

Reply:
OK, I will remove it from the implementations.